[suspendmanager] Protect floor designation from being erased (#725)
DFHack/scripts: docs/suspendmanager.rst

    items temporarily in the way, or worker dwarves getting scared by wildlife
- suspend construction jobs that would prevent a dwarf from reaching an adjacent
    construction job, such as when building a wall corner.
- suspend construction jobs on top of a smoothing, engraving or track carving

>   job. This prevent the construction job to be completed first, which would
>   erase the other

Reviewer: ```suggestion
  designation. This prevents the construction job from being completed first,
  which would erase the other.
```

Author: ```suggestion
  designation. This prevents the construction job from being completed first,
  which would erase the designation.
```
(using "designation" both times)
